rapthor/lib/screen.py

```diff
@@ -397,7 +412,7 @@ def get_memory_usage(self, cellsize_deg):
         yimsize = int(self.width_dec / cellsize_deg)  # pix
         test_array = np.zeros([1, len(self.freqs_ph), len(self.station_names), 4,
                                yimsize, ximsize])
-        mem_per_timeslot_gb = test_array.nbytes/1024**3 * 10  # include factor of 10 overhead
+        mem_per_timeslot_gb = test_array.nbytes/1024**3 / 10  # include factor of 10 overhead
```

Comment:
This change will reduce `mem_per_timeslot_gb` with a factor of 10, instead of increase it with a factor of 10. Is that intended?

Reply:
Yes, in my testing it uses a lot less memory than before. I'm not sure why (it could be the way the memory usage is estimated is not really right, but with the factor of 100 reduction, it does now seem to give a reasonable estimate)